tests/utils/test_utils.py

```diff
@@ -117,6 +119,18 @@ def test_readfile_binary(self):
         assert content.splitlines()[0] == b"Hello World!"
 
 
+@pytest.mark.usefixtures('freezer')
+class TestResourceFilename:
+
+    """Test resource_filename."""
+
+    def test_resource_filename(self):
+        """Read a test file."""
+        filename = utils.resource_filename(os.path.join('utils', 'testfile'))
+        with open(filename, 'r', encoding='utf-8') as f:
+            assert f.read().splitlines()[0] == "Hello World!"
```

Comment:
Please change this to a simple function as it's only one test - i.e. remove the class and `self` and move the `usefixtures` decorator to the function.